src/main/resources/js/condec.api.js

```diff
@@ -106,6 +106,33 @@
 		}
 	};
 
+	/*
+	 * external references: view.condec.issue.module, view.condec.knowledge.page ..
+	 */
+	ConDecAPI.prototype.createDecisionKnowledgeElementAsJIRAIssueComment = function createDecisionKnowledgeElementAsJIRAIssueComment(
+			summary, description, type, idOfParentElement, callback) {
+		if (summary !== "") {
+			var jsondata = {
+				"projectKey" : projectKey,
+				"summary" : summary,
+				"type" : type,
+				"description" : description,
+				"id" : idOfParentElement
```

Comment:
This should also include the documentation location in order to decide in backend whether it is a JIRA issue or not. Currently, this only works for JIRA issues but we want to add new elements also from sentence objects.